index.node.ts

```diff
@@ -8,8 +8,7 @@ import {
 } from './fabric';
 import { FabricObject } from './src/shapes/Object/Object';
 
-// TODO: move back to default values when refactoring to method
-FabricObject.prototype.objectCaching = false;
+FabricObject.ownDefaults.objectCaching = false;
```

Comment:
This can reside in Object `getDefaultValues` by querying env...

Reply:
oh i can do that yes

Reply:
i can't because we removed node from env.
This seems a good way to do it, in node, this default is configured differently.